src/core/namespace.c

```diff
@@ -83,7 +83,6 @@ static const TargetMount protect_kernel_tunables_table[] = {
         { "/proc/sysrq-trigger",        READONLY,       true  },
         { "/proc/latency_stats",        READONLY,       true  },
         { "/proc/mtrr",                 READONLY,       true  },
-        { "/proc/apm",                  READONLY,       true  },
```

Comment:
well, but I am not sure we should protect something this old. I mean, this is tech last updated 1996. According to the wikipedia page:

> Advanced power management (APM) is an API developed by Intel and Microsoft and released in **1992** which enables an operating system running an IBM-compatible personal computer to work with the **BIOS** to achieve power management.
> Revision 1.2 was the last version of the APM specification, released in **1996**. ACPI is intended as the successor to APM. **Microsoft dropped support for APM in Windows Vista**. The Linux kernel still mostly supports APM, with the **last fully functional APM support shipping in 3.3**.

(emphasis mine)

https://en.wikipedia.org/wiki/Advanced_Power_Management

I mean, it's fine if we protect slightly exotic stuff, but kernel 3.3 is not among the kernels we even support.

Let's not add new code that protects legacy crap. Let's not turn systemd into a collection of references to obsolete technology, in particular as on the systems we claim to support according to README the APM support **can't** work correctly anyway...